src/lib/components/ILSMenu/ILSMenu.js

```diff
@@ -149,7 +149,12 @@ class ILSMenu extends Component {
                   <div>
                     <EnvironmentLabel
                       pointing="left"
-                      style={{ marginLeft: '1.6em' }}
+                      small
+                      style={{
```

Comment:
please do not hardcode styling in the components, we have the theme less files for that